src/urls.py

```diff
@@ -0,0 +1,27 @@
+from django.urls import path
+
+from . import tasks
+
+urlpatterns = [
+    path('', tasks.index, name='index'),
+
+
+    path('task1', tasks.Task1.as_view(), name='task1'),
+    path('execute_task1', tasks.execute_task1, name='ExecuteTask1'),
+
+    path('task2', tasks.Task2.as_view(), name='task2'),
+    path('execute_task1', tasks.execute_task1, name='ExecuteTask2'),
```

Comment:
I think here it should be execute_task2. isn't it?

Reply:
Done